pkg/analysis/passes/binarypermissions/binarypermissions.go

```diff
@@ -0,0 +1,99 @@
+package binarypermissions
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/fs"
+	"os"
+	"path/filepath"
+
+	"github.com/grafana/plugin-validator/pkg/analysis"
+	"github.com/grafana/plugin-validator/pkg/analysis/passes/archive"
+	"github.com/grafana/plugin-validator/pkg/analysis/passes/metadata"
+)
+
+var (
+	binaryExecutableFound       = &analysis.Rule{Name: "binary-executable-found", Severity: analysis.Error}
+	binaryExecutablePermissions = &analysis.Rule{Name: "binary-executable-permissions", Severity: analysis.Error}
+)
+
+var REQUIRED_PERMISSIONS = fs.FileMode.Perm(0755)
```

Comment:
The Go convention does not use snake case, use camel case is used instead, even if it acts like a constant. This should become:

```go
var requiredPermissions = fs.FileMode.Perm(0755)
```